src/components/Contacts/CheckBox/CheckBox.tsx

```diff
@@ -0,0 +1,37 @@
+import { Checkbox, CheckboxProps, styled } from '@material-ui/core';
+import React from 'react';
+import theme from '../../../theme';
+
+export enum CheckBoxState {
+  'unchecked',
+  'checked',
+  'partial',
+}
+
+interface Props {
+  state?: CheckBoxState;
+  onClick?: () => void;
+}
+
+const StyledCheckBox = styled(Checkbox)(({ checked }: CheckboxProps) => ({
+  width: '18px',
+  height: '18px',
+  color: checked ? theme.palette.primary.dark : theme.palette.secondary.dark,
+}));
+
+export const CheckBox: React.FC<Props> = ({
```

Comment:
Is naming wrapper like components the same as Material UI a good practice? Just curious, I've tried to make the names different but haven't ran into needing the same name.

Reply:
Idk that I have a strong preference. ES6 imports make it clear where the component is imported from. But I could still see there being some ambiguity until you look up the import.

Reply:
Technically, Material UI has `Checkbox`, this is `CheckBox`... just kidding I'll change it, it can still get confusing.